packages/bpk-component-rating/package-lock.json

```diff
@@ -0,0 +1,279 @@
+{
+	"name": "bpk-component-rating",
+	"version": "0.0.20",
```

Comment:
Remove this line or it will just cause headaches later on 😆

Reply:
Good spot!